apps/charterafrica/src/payload/blocks/Datasets.js

```diff
@@ -0,0 +1,191 @@
+import { array } from "payload/dist/fields/validations";
+
+const sortOptions = [
+  "metadata_created desc",
+  "metadata_created asc",
+  "name asc",
+  "name desc",
+  "relevance desc",
+  "relevance asc",
+  "metadata_modified desc",
+  "metadata_modified asc",
+  "author asc",
+  "author desc",
+];
+
+const Datasets = {
+  slug: "datasets",
+  fields: [
+    {
+      label: {
+        en: "Sort Options",
+        fr: "Options de tri",
+        pt: "Opções de classificação",
+      },
+      type: "collapsible",
+      fields: [
+        {
+          name: "sortOptions",
+          type: "array",
+          minRows: 1,
+          label: {
+            en: "Options",
```

Comment:
```suggestion
            en: "Sort Order",
```
or Sort Options, etc.